workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/README.md

```diff
@@ -0,0 +1,184 @@
+# Multi Source Security Viewer
+
+This plugin enables you to visualize pipeline security information from multiple sources.
+
+## Setting up the plugin
+
+### For adminstrators
+
+#### Prerequisites
+
+1. Install the CI providers you want the plugin to handle the security scan outputs from.
+   The supported CI's are:
+
+- [Jenkins](../../../jenkins/plugins/jenkins-backend/README.md)
+- [Github Actions](../../../github-actions/plugins/github-actions/README.md)
+- [Gitlab CI](https://github.com/immobiliare/backstage-plugin-gitlab?tab=readme-ov-file#setup)
+
+#### Permission Framework Support
+
+The MSSV plugin has support for the permission framework.
```

Comment:
```suggestion
The Multi Source Security Viewer (MSSV) plugin has support for the permission framework.
```

If you are going to use an acronym it's best to use the full name with it in brackets that way the read has a point of reference.

Reply:
Updated